docs/stdlib/set.rst

```diff
@@ -193,20 +196,21 @@ Sets
 .. eql:operator:: coalesce: optional anytype ?? set of anytype \
                               -> set of anytype
 
-    Coalesce.
+    Performs coalescence between :eql:type:`anytype`.
```

Comment:
```suggestion
    Returns the second argument if the first argument is an empty set.
```
Less jargon is probably better here.

Reply:
I agree with avoiding jargon.

I'm trying to stick with "produce" for the result of an operator and "operand" for what's being operated on. Also, I'm concerned that this rewrite doesn't state what happens if the first argument isn't an empty set and that isn't necessarily obvious. That said, do you think this works?

```rest
    Produces the first of its operands that is not an empty set.
```

Reply:
I thought about this when making the PR and kept the technical jargon as it may be better to use specific words for conciseness, but I don't mind us removing it in favor of something more easily digestible for readers